checks/evaluation/permissions/permissions.go

```diff
@@ -99,77 +143,215 @@ func applyScorePolicy(results *checker.TokenPermissionsData, c *checker.CheckReq
 				loc.Snippet = newStr(r.File.Snippet)
 			}
 		}
-
 		text, err := createText(r)
 		if err != nil {
-			return checker.MinResultScore, err
+			return findings, err
 		}
 
-		msg, err := createLogMsg(r.LocationType)
+		f, err := createFinding(r.LocationType, text, loc)
 		if err != nil {
-			return checker.InconclusiveResultScore, err
+			return findings, err
 		}
-		msg.Finding = msg.Finding.WithMessage(text).WithLocation(loc)
+
 		switch r.Type {
-		case checker.PermissionLevelNone, checker.PermissionLevelRead:
-			msg.Finding = msg.Finding.WithOutcome(finding.OutcomePositive)
-			dl.Info(msg)
-		case checker.PermissionLevelUnknown:
-			dl.Debug(msg)
+		case checker.PermissionLevelNone:
+			f = f.WithOutcome(finding.OutcomePositive)
+			f = f.WithValues(map[string]int{
+				"PermissionLevel": int(PermissionLevelNone),
+			})
+		case checker.PermissionLevelRead:
+			f = f.WithOutcome(finding.OutcomePositive)
+			f = f.WithValues(map[string]int{
+				"PermissionLevel": int(PermissionLevelRead),
+			})
 
+		case checker.PermissionLevelUnknown:
+			f = f.WithValues(map[string]int{
+				"PermissionLevel": int(PermissionLevelUnknown),
+			})
```

Comment:
should this have a `finding.OutcomeError` or similar? by default `createFinding` creates a finding with `finding.OutcomeNegative`